### PR TITLE
Profile page: Add tags,edit option and ellipsis for extra info button with in sidebar

### PR DESCRIPTION
**Pull request by CleverFool77**
Fixes https://github.com/publiclab/plots2/issues/5891

Screenshot from 2019-07-29 15-38-20

profiles-debar

### CODE REVIEW

**Comment by CleverFool77** [comment]
Hi @jywarren This PR seems ready, Though I'll adding refinements to sidebar including information after toggle show in next PR. Should this be merged ? or something else is still left for now ?

**Comment by CleverFool77** [comment]
Hi @gauravano This PR is ready for review.
Thanks !!

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -278,6 +289,18 @@
     })();
 </script>
 
+<script>
```
Comment (gauravano) [inline app/views/users/profile.html.erb]: Please append this script blob in the above one only. Also, please indent it.

File: app/views/users/profile.html.erb [hunk 2]
```diff
@@ -45,186 +45,197 @@
...
-      <li class="mt-3"><h5>@<%= @profile_user.name %> <%= @profile_user.new_contributor %> asked</h5></li>
-      <li><h5><%= @profile_user.note_count %><a href='/notes/author/<%= @profile_user.name %>'> research notes</a></h5></li>
-      <li><h5><%= @profile_user.revisions.count %> wiki edits</h5></li>
-      <li><h5><a href = "/tag/question:*/author/<%= params[:id] %>"><%= Node.questions.where(status: 1, uid: @profile_user.id).length %> questions </a></h5></li>
-      <li><h5><%= Comment.where(uid: @profile_user.id).count %> comments</h5></li>
-      <li><h5><a href = "/tag/activity:*/author/<%= params[:id] %>"><%= @count_activities_posted %> activities posted </a></h5></kli>
-      <li><h5><a href = "/tag/replication:*/author/<%= params[:id] %>"><%= @count_activities_attempted %> activities attempted</a> </h5></li>
-    </ul>
+  <div class="mt-5 text-center">
+    <a class="btn btn-outline-secondary" id="tagss">Tags</a>
```
Comment (gauravano) [inline app/views/users/profile.html.erb]:
```suggestion
    <a class="btn btn-outline-secondary" id="tags-section">Tags</a>
```

File: app/views/users/profile.html.erb [hunk 3]
```diff
@@ -45,186 +45,197 @@
...
+    <% if current_user && current_user.uid == @profile_user.uid %>
+    <a href="/profile/<%= @profile_user.name %>/edit" class="btn btn-outline-secondary"><%= t('users.profile.edit_profile') %></a>
```
Comment (gauravano) [inline app/views/users/profile.html.erb]: Indentation

File: app/views/users/profile.html.erb [hunk 4]
```diff
@@ -45,186 +45,197 @@
...
-      <div id="follow_form" class="col-md-offset-4">
-        <% if current_user.following?(@profile_user) %>
-          <a id="profile-unfollow-button" class="btn btn-primary" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>">Unfollow</a>
-        <% else %>
-          <a id="profile-follow-button" class="btn btn-primary" data-method="post" data-remote="true" href="/relationships" data-params="followed_id=<%= @profile_user.id %>">Follow</a>
-        <% end %>
-      </div>
-      <br />
+
+  <div id="all-info">
```
Comment (gauravano) [inline app/views/users/profile.html.erb]:
```suggestion
  <div id="social-links">
```

**Comment by gauravano** [comment]
Great work @CleverFool77 :tada:. Just some minor changes.

**Comment by CleverFool77** [comment]
> Also, can you please change the design of access token and test digest button, use same design as other? -
> 
> Screenshot from 2019-08-02 00-40-15

Hi @gauravano I'll do the changes you mentioned above but I'm not sure about this change. As this design was already there in the previous design. That specific color different from others  being given to some buttons.

**Comment by CleverFool77** [comment]
Hi @gauravano I just updated the changes in PR you have mentioned in review.
Thanks !!!

**Comment by CleverFool77** [comment]
Build failed :thinking:. What happened to travis. Its happening again and again.

**Comment by CleverFool77** [comment]
Hi @jywarren @gauravano , I was waiting for one of the PR to get merged. Now that PR for shifting of username is merged. So I have updated the latest changes in this PR.
Screenshot attached - 
Screenshot from 2019-08-03 12-36-15

Regarding the Info of user , that is `posted`, `commented` stuff in second column. Do You want it to centre aligned or from left ?

**Comment by jywarren** [comment]
Center looks good! Although we dont specify in style guide. I could go either way. 

So all that's left here is the chromedriver issue were working on? Nothing else in system tests is failing? Thanks!

**Comment by gauravano** [comment]
@CleverFool77 please rebase your PR. 

And, regarding
> Hi @gauravano I'll do the changes you mentioned above but I'm not sure about this change. As this design was already there in the previous design. That specific color different from others being given to some buttons.

yeah, I know, but, we want to have consistent design for all buttons so just asked as you were changing code in the same file. Maybe FTO can be opened for this issue?

Thanks!

**Comment by CleverFool77** [comment]
> @CleverFool77 please rebase your PR.
> 
> And, regarding
> 
> > Hi @gauravano I'll do the changes you mentioned above but I'm not sure about this change. As this design was already there in the previous design. That specific color different from others being given to some buttons.
> 
> yeah, I know, but, we want to have consistent design for all buttons so just asked as you were changing code in the same file. Maybe FTO can be opened for this issue?
> 
> Thanks!

Hi @gauravano 

> @CleverFool77 please rebase your PR.
> 
> And, regarding
> 
> > Hi @gauravano I'll do the changes you mentioned above but I'm not sure about this change. As this design was already there in the previous design. That specific color different from others being given to some buttons.
> 
> yeah, I know, but, we want to have consistent design for all buttons so just asked as you were changing code in the same file. Maybe FTO can be opened for this issue?
> 
> Thanks!

Ohkk Cool !! lets discuss with others and after that I'll open a First-timers only issue for it.
Thanks !!

**Comment by gauravano** [comment]
Hey @CleverFool77, how about justifying the tags:

Screenshot from 2019-08-07 01-12-12

**Comment by gauravano** [comment]
Merged :tada: . Thanks for the great work @CleverFool77!

**Pull request merged**